opennebula/resource_opennebula_virtual_network.go

```diff
@@ -751,6 +794,8 @@ func resourceOpennebulaVirtualNetworkRead(d *schema.ResourceData, meta interface
 	d.Set("gname", vn.GName)
 	d.Set("bridge", vn.Bridge)
 	d.Set("physical_device", vn.PhyDev)
+	d.Set("reservation_vnet", vn)
```

Comment:
this should contains an ID, at first glance it seems to me that's the right place to use `vn.ParentNetworkID` field